docs/multi-instance.md

```diff
@@ -8,13 +8,89 @@ See [554#issuecomment-496804929](https://github.com/elastic/ansible-elasticsearc
 
 If you have single-instances hosts and want to upgrade from previous versions of the role:
 
+### Procedure with datas move
+
+This procedure will allow you to move your datas to the new standard paths (see [#581](https://github.com/elastic/ansible-elasticsearch/issues/581)):
+
+1. Stop Elasticsearch before the migration
+
+2. Migrate your datas to the new standard paths:
+```
+# mv /etc/elasticsearch/${ES_INSTANCE_NAME}/* /etc/elasticsearch/ && rm -fr /etc/elasticsearch/${ES_INSTANCE_NAME}/
+mv: overwrite '/etc/elasticsearch/elasticsearch.keystore'? y
+# mv /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/lib/elasticsearch/ && rm -fr /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# ls /var/lib/elasticsearch/
+nodes
+# mv /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/log/elasticsearch/ && rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+# rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+```
+
+3. Update playbook (remove `es_conf_dir`, `es_data_dirs`, `es_log_dir`, `es_pid_dir` and `es_instance_name` variables)
+
+4. Deploy ansible-role
+
+5. After ansible-role new deployment, you can do some cleanup of old Init file and Default file:
+
+Example:
+```
+$ systemctl stop elasticsearch
+$ mv /etc/elasticsearch/${ES_INSTANCE_NAME}/* /etc/elasticsearch/ && rm -fr /etc/elasticsearch/${ES_INSTANCE_NAME}/
+mv: overwrite '/etc/elasticsearch/elasticsearch.keystore'? y
+$ mv /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/lib/elasticsearch/ && rm -fr /var/lib/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+$ ls /var/lib/elasticsearch/
+nodes
+$ mv /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/* /var/log/elasticsearch/ && rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
+$ rm -fr /var/log/elasticsearch/${INVENTORY_HOSTNAME}-${ES_INSTANCE_NAME}/
```

Comment:
This `rm` happens in the previous line as well.

Reply:
@Conky5 fixed